.terraform-version

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
this should be symlinked to the component, see the example in https://github.com/hmcts/cnp-azuredevops-libraries#required-terraform-folder-structure